lib/network/modules/InteractionHandler.js

```diff
@@ -705,6 +708,28 @@ class InteractionHandler {
       this.body.emitter.emit('hidePopup');
     }
   }
+
+
+  /**
+   * Ensure zoom min/max values are acceptable
+   * If they aren't, remove them from `options`
+   * @private
+   */
+  _checkRemoveZoomMinMax(options) {
+    const { zoomMin, zoomMax } = options;
+
+    const notValid = value => (
+      !util.isNumber(value) || Number.isNaN(value) || value < 0
+    );
+
+    if (notValid(zoomMin) || zoomMin > zoomMax) {
```

Comment:
Why can't they both be the same? Wouldn't that just pin the zoom level to a single value? What's wrong with that?

Reply:
I'm not clear on what you mean here. Do you mean that the OR condition isn't in the right place (e.g. you want `notValid` to also check for the `zoomMin > zoomMax` condition)?

Taking into consideration your other comment above, this is what I have now. Can you explain a bit more about the "Why can't they both be the same?" comment?

```javascript
this.defaultOptions = {
  ...
  zoomMin: 0.00001,
  zoomMax: 10
  ...
};

const isValid = value => (
  util.isNumber(value)
  && value > this.defaultOptions.zoomMin
  && value < this.defaultOptions.zoomMax
);

if (!isValid(zoomMin) || zoomMin > zoomMax) {
  delete options.zoomMin;
}

if (!isValid(zoomMax) || zoomMax < zoomMin) {
  delete options.zoomMax;
}
```

Reply:
Definitely not. I mean why can't min and max zoom be the same.

Reply:
Ah, I see. Okay, will factor that in.